src/data/faq.json

```
@@ -293,12 +293,15 @@
                         ]
                     },
                     {
-                        "title": "Can I use a complete vaccination status in the app as a travel permission?",
+                        "title": "Can I use my vaccination certificate in the app as a travel permission?",
                         "anchor": "vac_cert_privileges_travel",
                         "active": true,
                         "textblock": [
-                            "For example, whether you are allowed to travel with completed vaccination protection, depends on the regulations of the respective country.",
-                            "Among other things, you can use this   <a href='https://reopen.europa.eu/en' target='_blank' rel='noopener noreferrer'>tool</a> in order to plan your journey within the EU."
+                            "The digital certificate of vaccination of the EU can not only be used in Germany. The digital certificate of vaccination of the EU issued in Germany can also be regarded as proof of vaccinations in other EU countries if the respective EU member state allows digital proof. The same applies to recovery certificates and test certificates. From Corona-Warn-App version 2.6, the validity of your certificate can also be checked within the app before departure.",
+                            "In addition, the respective EU member states generally also check the corresponding certificate with their own verifier apps. The data can be read out regardless of the language of the certificate. It is therefore irrelevant in which language your certificate is displayed on your device.",
+                            "The digital certificiate of vaccination of the EU can be presented as a printed QR code or as a scan in the Corona-Warn-App or CovPass-App. The known proof of vaccination in the form of the yellow vaccination booklet remains valid. The new digital certificate of vaccination of the EU for a COVID-19 vaccination is merely a user-friendly and trustworthy addition to the yellow vaccination booklet that can be easily checked.",
+                            "Please note that sufficient effectiveness of vaccination protection does not occur until 14 days after the last necessary vaccination.",
+                            "Summarized information about the corona measures and travel restrictions applicable in the European countries can be found on this page: <a href='https://reopen.europa.eu/de'  target='_blank' rel='noopener noreferrer'>https://reopen.europa.eu/de</a>. The update is the responsibility of the respective countries. There you will also find information on what personal information and documents you must present upon entry. Data processing is subject to the regulations of the respective country of entry. Details can be found on the national websites of the respective EU countries, which are linked on the above-mentioned page."

```

```suggestion
                            "Summarized information about the corona measures and travel restrictions applicable in the European countries can be found on this page: <a href='https://reopen.europa.eu/en'  target='_blank' rel='noopener noreferrer'>https://reopen.europa.eu/en</a>. The update is the responsibility of the respective countries. There you will also find information on what personal information and documents you must present upon entry. Data processing is subject to the regulations of the respective country of entry. Details can be found on the national websites of the respective EU countries, which are linked on the above-mentioned page."
```
Change to English-language site